templates-loop/content-card.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Post rendering content according to caller of get_template_part
+ *
+ * @package uds-wordpress-theme
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+// get the thumbnail alt text.
+$thumbnail_id = get_post_thumbnail_id( $post->ID );
+$alt_text = get_post_meta( $thumbnail_id, '_wp_attachment_image_alt', true );
+
+?>
+
+<div class="col col-12 col-md-6 col-lg-4 mb-4">
+
+	<div class="card card-story" id="post-<?php the_ID(); ?>" >
+
+		<?php
+		if ( has_post_thumbnail() ) {
+			echo '<img class="card-img-top" src="' . get_the_post_thumbnail_url( $post->ID, 'large' ) . '" alt="' . $alt_text . '"/>';
```

Comment:
Is the large image preset the best preset to use for Card images? Given the size of the rendered image for Cards, medium is probably more efficient. We need to be conscientious about the file size we choose for rendered assets, like images. Choosing an excessively large image just wastes bandwidth, slowing down page load times and affecting our SEO and PageRank scores.

Reply:
Addressed. See comments below. :-)